src/templates/react/Content/public/index.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html>
+<head>
+  <meta http-equiv='Content-Type' content='text/html; charset=utf-8'>
+  <title>Elmish  Fable App</title>
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css" />
+  <script src="https://cdn.polyfill.io/v2/polyfill.js?features=default,fetch"></script>
```

Comment:
Good point, @et1975, I didn't notice this line. Didn't know about polyfill.io either. Looks very interesting, as it only provides the polyfills needed for the current browser. It also provides `fetch` compatibility, which doesn't come in core-js. Maybe we could just disable core-js in the babel-transform-runtime plugin and use this instead?